src/transformers/modeling_bart.py

```diff
@@ -578,6 +588,10 @@ def forward(
             if output_attentions:
                 all_self_attns += (layer_self_attn,)
 
+        if self.variant == "prelayernorm":
+            x = self.layernorm_embedding(x)
+
+        # Everything below here is book keeping
```

Comment:
I don't understand this comment @mariamabarham - what does that mean "book keeping"?

Reply:
maybe @sshleifer ? I saw it but didn't really mind it

Reply:
What I mean is that no more tensor ops, below here we just get outputs into the correct format.
Should probably say: no more tensor ops below here, just boilerplate or some such.